app/editor/AlignMentButtonContainer.tsx

```diff
@@ -0,0 +1,56 @@
+import {
+  Tooltip,
+  TooltipContent,
+  TooltipProvider,
+  TooltipTrigger,
+} from "@/components/ui/tooltip";
+import { AlignCenter, AlignJustify, AlignLeft, AlignRight } from "lucide-react";
```

Comment:
I was talking about all these icons. We might forget some of these after merging because there are a lot, possible to move to icons.tsx please?